.github/workflows/test.yml

```diff
@@ -43,12 +43,8 @@ jobs:
       run: |
         choco install vcpython27 -f -y
 
-    - name: Get PR labels
-      id: pr-labels
-      uses: joerick/pr-labels-action@v1.0.6
```

Comment:
@joerick, this seems like an interesting/important consequence towards your custom action/repository?

Reply:
Maybe this pre-dates the syntax for accessing labels?

Reply:
No idea why it wasn't working, it [still works on my repo](https://github.com/joerick/pr-labels-action/pull/5/checks?check_run_id=1717049797). Labels can be missed depending on when they're added - they're snapshotted when the build was triggered - perhaps the label was added after the build was triggered?

Reply:
I restarted the build after adding the label, it should have been picked up? Though I didn't try closing and reopening.

Is there a benefit to the action over just using an if?